addons/dragging/functions/fnc_carryObjectPFH.sqf

```diff
@@ -31,6 +31,25 @@ if !(_unit getVariable [QGVAR(isCarrying), false]) exitWith {
     [_idPFH] call CBA_fnc_removePerFrameHandler;
 };
 
+// Update mouse hint
+private _ctrlTextLMB = (uiNamespace getVariable [QEGVAR(interaction,mouseHint), displayNull]) displayCtrl 2420;
+if (
+    !isNull cursorObject
+    && {([ACE_player, cursorObject, ["isNotCarrying"]] call EFUNC(common,canInteractWith))}
+    && {
+        if (_target isKindOf "CAManBase") then {
+            ([] isNotEqualTo ([cursorObject, 0, true] call EFUNC(common,nearestVehiclesFreeSeat)))
+            && {([_target, cursorObject] call EFUNC(interaction,getInteractionDistance)) < MAX_LOAD_DISTANCE}
+        } else {
+            [_target, cursorObject] call EFUNC(cargo,canLoadItemIn)
+        }
+    }
+) then {
+    _ctrlTextLMB ctrlSetText localize ELSTRING(Cargo,loadObject);
+} else {
+    _ctrlTextLMB ctrlSetText localize LSTRING(Drop);
+};
```

Comment:
Could maybe do what `refuel` does instead of directly manipulating the mouse hint UI. https://github.com/acemod/ACE3/blob/f67957cdde931ee7cb1bffbbb0457efa08640c15/addons/refuel/functions/fnc_startNozzleInHandsPFH.sqf#L93-L113

Reply:
That's an elegant approach! I'll give it a shot.
I based what I did on https://github.com/acemod/ACE3/blob/f67957cdde931ee7cb1bffbbb0457efa08640c15/addons/explosives/functions/fnc_setupExplosive.sqf#L121-L131